lib/jsdom/living/blob.js

```diff
@@ -0,0 +1,76 @@
+"use strict";
+
+const blobSymbols = require("./blob-symbols");
+
+class Blob {
+  constructor() {
+    if (!(this instanceof Blob)) {
+      throw new TypeError("DOM object constructor cannot be called as a function.");
+    }
+    const parts = arguments[0];
+    const properties = arguments[1];
+    if (arguments.length > 0) {
+      if (!parts || typeof parts !== "object" || parts instanceof Date || parts instanceof RegExp) {
+        throw new TypeError("blobParts should be nor a non-objects nor Dates nor RegExps nor null");
```

Comment:
Nit: a better error message would be "Blob parts must be objects that are not Dates or RegExps".